content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -403,6 +410,31 @@ In the past, the annotation `volume.beta.kubernetes.io/mount-options` was used i
 of the `mountOptions` attribute. This annotation is still working, however
 it will become fully deprecated in a future Kubernetes release.
 
+### Node Affinity
+
+{{< note >}}
+**Note:** You do not need to set this field. It is automatically populated for aws, gce and azure volume block types. You need to explicitly set this in case of local volumes. Most other volume types do not need this field.
```

Comment:
A few points:

* "In most cases, you do not need to set this field".
* For aws, gce, azure, local, link to the descriptions on the volumes.md page

Reply:
updated the note. Added the relevant links